test/autorest.csharp.test.csproj

```diff
@@ -36,6 +36,7 @@
     <PackageReference Include="System.Reflection.TypeExtensions" Version="4.3.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="2.3.1" />
     <PackageReference Include="System.Net.Http" Version="4.3.2" />
+    <PackageReference Include="System.Collections.Concurrent" Version="4.3.0" />

```

Comment:
maybe I missed latest developments, but is there still a collection required in the way we model enums now?

Reply:
hmm... that's true, let me see if I can get rid of it